src/main/scala/dpla/ingestion3/mappers/Mapper.scala

```diff
@@ -316,7 +317,8 @@ class JsonMapper extends Mapper[JValue, JsonMapping] {
           title = validatedTitle,
           `type` = validatedType
         ),
-        messages = msgCollector.getAll()
+        messages = msgCollector.getAll(),
+        originalId = mapping.originalId(document)
```

Comment:
swapping lines 320 and 321 is the only other tweak I would make. Collecting messages from mapping should be the very last thing that happens otherwise we might lose something.

Reply:
:+1: